package.json

```diff
@@ -6,7 +6,7 @@
     "clean": "rimraf dist",
     "tsc": "tsc",
     "build": "npm run clean && npm run tsc",
-    "start": "npm run build && node dist/index.js",
+    "start": "node dist/index.js",
```

Comment:
I'm also not sure if this will affect anything. @Jawnnypoo would know better than me since he setup the typescript env.

If it does affect anything, simply add another configuration and just call it `dockerstart` or equivalent.